### PR TITLE
RELATED: BB-583 Remove footer space in view mode when totals hidden

### PR DESCRIPTION
**Pull request by tomassatinsky**

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/505/ : SUCCESS in 16s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/308/ : FAILURE in 1m 13s

File: src/Table/utils/test/header.spec.js [hunk 1]
```diff
@@ -77,7 +77,7 @@ describe('Table utils - Header', () => {
     });
 
     describe('isHeaderAtDefaultPosition', () => {
-        it('should return true if header is scrolled below zero sticky header offset', () => {
+        it('should retu§rn true if header is scrolled below zero sticky header offset', () => {
```
Comment (petrjanu) [inline src/Table/utils/test/header.spec.js]: §

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/506/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/309/ : SUCCESS in 1m 43s

File: src/Table/utils/test/header.spec.js (hunk 1 above)
Comment (tomassatinsky) [reply src/Table/utils/test/header.spec.js]: thanks, fixed

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/508/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/311/ : SUCCESS in 1m 47s

**Comment by tomassatinsky** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/511/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/315/ : SUCCESS in 5m 21s

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-indigo-visualizations-storybook-zuul-docker/107/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/512/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-pull-request-dispatcher/263/ : SUCCESS in 5m 20s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-merge-compliance-check/100/ : SUCCESS in 13s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/316/ : SUCCESS in 2m 11s

**Pull request merged**